<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CougarBackup</class>
 <widget class="QWidget" name="CougarBackup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>340</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>770</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>770</width>
    <height>500</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>770</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CougarBackup</string>
  </property>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_number">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Number:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_diskLocation">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>50</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Disk Location:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>230</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>230</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>411</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_source">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>91</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="lineWrapMode">
    <enum>QTextEdit::NoWrap</enum>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_destination">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>121</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_source">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>91</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Source:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_destination">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>121</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Destination:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>10</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_number">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_browse">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>44</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_folders">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>30</y>
     <width>331</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_folders">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Excluded Folders</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_excluded">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>48</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Excluded Folders:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exclude">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>43</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_update">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>151</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string extracomment="Check this if running rsync again on the same backup. Will backup with the -u option"/>
   </property>
   <property name="whatsThis">
    <string extracomment="Check this if running rsync again on the same backup. Will backup with the -u option"/>
   </property>
   <property name="text">
    <string>Update Option</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_windows">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>181</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Windows OS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_config">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select Config Folder</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_output">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>750</width>
     <height>301</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_outFinished">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>314</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Close Window</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>270</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Clear Exclusions</string>
   </property>
  </widget>
  <zorder>label_name</zorder>
  <zorder>label_number</zorder>
  <zorder>label_diskLocation</zorder>
  <zorder>pushButton_start</zorder>
  <zorder>pushButton_cancel</zorder>
  <zorder>line</zorder>
  <zorder>textBrowser_source</zorder>
  <zorder>textBrowser_destination</zorder>
  <zorder>label_source</zorder>
  <zorder>label_destination</zorder>
  <zorder>lineEdit_name</zorder>
  <zorder>lineEdit_number</zorder>
  <zorder>pushButton_browse</zorder>
  <zorder>textBrowser_folders</zorder>
  <zorder>label_folders</zorder>
  <zorder>label_excluded</zorder>
  <zorder>pushButton_exclude</zorder>
  <zorder>checkBox_update</zorder>
  <zorder>checkBox_windows</zorder>
  <zorder>pushButton_config</zorder>
  <zorder>pushButton_outFinished</zorder>
  <zorder>pushButton_clear</zorder>
  <zorder>textBrowser_output</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
